### ISSUE TITLE
Web Apps backup feature does not backup Ghost db

### ISSUE THREAD

**Issue by smereczynski** [felixrieseberg/Ghost-Azure#43]
If You configure backups feature on Azure Web Apps for site with Ghost, the ghost.db file will be not backed up because of some locks on disk C.

**Comment by RyanONeill1970** [comment]
Yes, this has just caused me a severe issue. My fault, I should have backed up manually first.

Azure didn't backup and deploy from zip and the updater app all failed, my content is now gone.

**Comment by RyanONeill1970** [comment]
Found this, would it be possible to integrate this into the main build?
https://tomssl.com/2016/09/12/backup-your-sqlite-database-with-zero-downtime-when-running-ghost-in-azure-web-apps/

**Comment by TomChantler** [comment]
Thanks Ryan, I was intending to contact the team and mention this myself. I'm really sorry to hear you lost your data. That's rough (to say the least).

I'm actually in the process of making a better (and very simple) way of keeping Ghost completely up to date in Azure with zero downtime which will include a version of this WebJob (or something broadly similar). There'll be a follow-up blog post about it very soon.

**Comment by RyanONeill1970** [comment]
That's good. I did manage to recover enough to continue, as I say, my fault, I knew what I was doing when I chose not to copy the file before upgrading...
